app/src/main/scala/org/alephium/explorer/persistence/queries/HashrateQueries.scala

```diff
@@ -33,33 +34,42 @@ trait HashrateQueries extends CustomTypes {
   def getHashratesQuery(from: TimeStamp,
                         to: TimeStamp,
                         interval: Int): DBActionSR[(TimeStamp, BigDecimal)] = {
-    val fromInstant = Instant.ofEpochMilli(from.millis)
-    val toInstant   = Instant.ofEpochMilli(to.millis)
 
     sql"""
         SELECT timestamp, value
         FROM hashrates
-        WHERE interval_type = #$interval
-        AND timestamp >= '#${fromInstant.toString}'
-        AND timestamp <= '#${toInstant.toString}'
+        WHERE interval_type = $interval
+        AND timestamp >= $from
+        AND timestamp <= $to
```

Comment:
shall the condition be `[from, to)`, half close and half open interval?

Reply:
mmm again it's a matter of point of view :)  if I request hashrates from 08:00 to 14:00 I would expect to have both 08:00 and 14:00 in the data I receive.

Reply:
okay, then let's use close interval

Reply:
the main issue is that if one queries for several sequential interval, some hashrate would appear in two queries

Reply:
ah good point,  ok so let's say if you request  08:00 to 14:00, you'll get 8:00 and 13:00

Reply:
let's keep it like this for now and we'll see if someone complains